lightning/src/ln/channel.rs

```diff
@@ -5417,7 +5417,10 @@ impl<SP: Deref> Channel<SP> where
 	}
 
 	fn get_last_revoke_and_ack(&self) -> msgs::RevokeAndACK {
-		let next_per_commitment_point = self.context.holder_signer.as_ref().get_per_commitment_point(self.context.holder_commitment_point.transaction_number(), &self.context.secp_ctx);
+		debug_assert!(self.context.holder_commitment_point.transaction_number() <= INITIAL_COMMITMENT_NUMBER + 2);
```

Comment:
Trying to understand the `+ 2`. The bolts seem to read like our transaction number will never be > `INITIAL_COMMITMENT_NUMBER`, let me know what I'm missing here!

Reply:
ah oops, this should actually be `- 2`, to assert we have always advanced our commitment point twice before we ever call here

Reply:
Ah that makes sense lol